app/scripts/Annotations2dTrack.js

```diff
@@ -413,7 +413,7 @@ class Annotations2dTrack extends TiledPixiTrack {
 
     if (this.options.onSelect && !silent) {
       window[this.options.onSelect](uid);
-      pubSub.publish('annoSelected', uid);
+      pubSub.publish('annoSelected', uid); // eslint-disable-line no-undef
```

Comment:
```suggestion
    this.pubSub.publish('app.click', {
```
No need to ignore this case